contracts/src/v0.8/functions/dev/v1_X/ocr/OCR2Base.sol

```diff
@@ -332,17 +330,12 @@ abstract contract OCR2Base is ConfirmedOwner, OCR2Abstract {
         expectedNumSignatures = configInfo.f + 1;
       }
 
-      // solhint-disable-next-line custom-errors
-      require(rs.length == expectedNumSignatures, "wrong number of signatures");
-      // solhint-disable-next-line custom-errors
-      require(rs.length == ss.length, "signatures out of registration");
+      if (rs.length != expectedNumSignatures) revert ReportInvalid("wrong number of signatures");
+      if (rs.length != ss.length) revert ReportInvalid("signatures out of registration");
```

Comment:
This error message is confusing to me, what does it mean "out of registration" :) Maybe @stchrysa can help re-word.